syntax = "proto3";

package proto.ext.common;

message AckResponse {};

message ErrorResponse {
  string error_text = 1;
};
